ui/src/pages/backoffice/Series/SeriesDetails/__tests__/__snapshots__/SeriesDetails.test.js.snap

```diff
@@ -7,8 +7,7 @@ exports[`SeriesDetails tests should load the details component 1`] = `
   >
     <Error>
       <SeriesMetadata />
-      <SeriesDocuments />
-      <SeriesMultipartMonographs />
+      <SeriesLiterature />
```

Comment:
as I remember from our conversations. the Literature search was meant to be only for the fronstsite, I think for the librarians it should be still separated - it is clearer for managing